README.md

```diff
@@ -223,6 +223,10 @@ your IBM Cloud Cluster API Kubernetes cluster.
 
 Please refer to [Create a new woker node](docs/add_node.md) for further info.
 
+### How to use clusterctl image
+
+Please refer to [How to use clusterctl image](docs/how_to_use_clusterctl.md) for further info.
```

Comment:
Could you add simple introduction when someone need to read this document?
For trouble shooting and create worker nodes, it is clear from the title. But for clusterctl image, it is misunderstanding I think.

Reply:
Done.